scripts/validate

```diff
@@ -21,19 +21,37 @@ test -z "$failed"
 echo "Running: go fmt"
 test -z "$(go fmt ${PACKAGES} | tee /dev/stderr)"
 
-## validation: Benchmarks (cfgs) yaml files with yamllint
+## validation: Benchmarks (cfgs) with yamllint and kube-bench (dry-run) 
 echo "Running: yamllint against security-scan cfgs"
 
+# CFGS lists all benchmark directories such as cis-1.7
 CFGS="$(find package/cfg/ -mindepth 1 -type d -printf "%f\n")"
+
 FAILED_CFGS_YML=()
+FAILED_CFGS_KB=()
+
+# Loop through all benchmarks
 for cfg in ${CFGS}; do
 set -x
+# yammlint is configured with ../.yamllint.yaml, to catch any liniting errors for a given benchmark. If result isn't empty, FAILED_CFGS_YML is appended with the failing benchmark.
     if [ -n "$(yamllint -s package/cfg/$cfg)" ]; then
-        FAILED_CFGS_YML+=("$package/cfg/cfg")
+        FAILED_CFGS_YML+=("package/cfg/$cfg")
+    fi
+# kube-bench dry-run to detect any errors for a given benchmark's files - check 1.1 is common to all benchmarks and used to speed up the test. If kube-bench commands fails, FAILED_CFGS_KB is appended with the failing benchmark.
+    if kube-bench --config-dir package/cfg --config package/cfg/config.yaml --benchmark $cfg --check 1.1 --noremediations --noresults --nosummary --nototals; then
```

Comment:
```suggestion
    if kube-bench --config-dir package/cfg --config package/cfg/config.yaml --benchmark "${cfg}" --check 1.1 --noremediations --noresults --nosummary --nototals; then
```

Reply:
Thanks @pjbgf, I added the quotes as requested in the last commit, see [validate](https://github.com/rancher/security-scan/blob/d99188a9bdfe27d1e8da84a946e1824f6c4260f7/scripts/validate#L41).